nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/WEB-INF/partials/canvas/processor-configuration.jsp

```diff
@@ -108,6 +108,18 @@
                         </div>
                         <div class="clear"></div>
                     </div>
+                    <div class="setting">
```

Comment:
The Scheduling Strategy and the Run Schedule should probably remain closer together. Can you shift the Execution Node setting below the Concurrent Tasks and Run Schedule.